### PR TITLE
Dvclive pipeline transition

### PR DESCRIPTION
**Pull request by dberenbaum**
I think this is enough to close https://github.com/iterative/dvclive/issues/648 and for now seems to me like a better solution than any kind of auto-conversion of dvclive-only workflows to pipelines. The experience is still not completely smooth, but the goal for this PR is to build on https://github.com/iterative/dvc.org/pull/4795 and explain the potential friction points and how to solve them.

### CODE REVIEW

File: content/docs/dvclive/how-it-works.md [hunk 1]
```diff
@@ -109,22 +109,80 @@ with Git, in which case you can use
 
 ## Setup to Run with DVC
 
-You can create or modify the `dvc.yaml` file at the base of your repository (or
-elsewhere) to define a [pipeline](#setup-to-run-with-dvc) to run experiments
-with DVC or
-[customize plots](/doc/user-guide/experiment-management/visualizing-plots#defining-plots).
-A pipeline stage for model training might look like:
+To run experiments with DVC, define a <abbr>DVC pipeline</abbr> using
```
Comment (shcheklein) [inline content/docs/dvclive/how-it-works.md:112]: It would be great to have an explanation why it is better (to run with DVC)

**Review by shcheklein — APPROVED** [review]

File: content/docs/dvclive/how-it-works.md (hunk 1 above)
Comment (mnrozhkov) [reply content/docs/dvclive/how-it-works.md:112]: TBH we don't know whether it's better or not :)  It depends on the use case. But it makes sense to give an overview of possible scenarios where DVCLive or DVC is preferred.

**Review by mnrozhkov — APPROVED** [review]

File: content/docs/dvclive/how-it-works.md [hunk 2]
```diff
@@ -109,22 +109,80 @@ with Git, in which case you can use
...
+adding more heavyweight plots data to the cache:
+
+<toggle>
+<tab title="CLI">
+
+```cli
+$ dvc stage add --name train \
+  --deps data_dir --deps src/train.py \
+  --outs model.pt --outs dvclive/metrics.json \
+  --outs-no-cache dvclive/plots \
```
Comment (daavoo) [inline content/docs/dvclive/how-it-works.md:157]: Shouldn't this be:

```suggestion
  --outs model.pt --outs-no-cache dvclive/metrics.json \
  --outs dvclive/plots \
```

Comment (dberenbaum) [reply content/docs/dvclive/how-it-works.md:157]: Good catch!

File: content/docs/dvclive/how-it-works.md (hunk 1 above)
Comment (dberenbaum) [reply content/docs/dvclive/how-it-works.md:112]: My rationale here was that it's addressed in many other places (like in `dvclive/index`) and this page is more about the how than the why. Not a blocker for me either way, though, so let me see if I can come up with something to put here.

**Pull request merged**